services/challengedeployerservice/utils/helper.go

```diff
@@ -21,10 +21,18 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
+	g "github.com/sdslabs/katana/configs"
+)
+
```

Comment:
Why do we need a separate util package for the service? Just having a helper.go/util.go in a package is standard practice in go